homeassistant/components/device_tracker/ble_tracker.py

```diff
@@ -0,0 +1,90 @@
+"""Tracking for bluetooth devices."""
+import logging
+from datetime import timedelta
+
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.components.device_tracker import (
+    YAML_DEVICES,
+    CONF_TRACK_NEW,
+    CONF_SCAN_INTERVAL,
+    DEFAULT_SCAN_INTERVAL,
+    load_config,
+)
+import homeassistant.util as util
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['gattlib']
+
+BLE_PREFIX = 'BLE_'
+
+def setup_scanner(hass, config, see):
+    """Setup the BLE Scanner."""
```

Comment:
The Bluetooth tracker and the Bluetooth Low Energy tracker are different. The Bluetooth tracker can only detect Bluetooth devices, but not BLE, while the new module can only track Bluetooth Low energy devices. 
While booth seems to be the same, Bluetooth and Bluetooth Low Energy differ a lot.
That's why I don't want to call it "bluetooth_tracker_...". If you don't want abbreviations, I can rename it to "bluetooth_le_tracker" or "bluetooth_low_energy_tracker". Personally I think both names a bit long.